src/server/scripts/World/npcs_special.cpp

```diff
@@ -1867,62 +1867,61 @@ class npc_pet_trainer : public CreatureScript
 ## npc_experience
 ######*/
 
-#define EXP_COST                100000 //10 00 00 copper (10golds)
-#define GOSSIP_TEXT_EXP         14736
-#define GOSSIP_XP_OFF           "I no longer wish to gain experience."
```

Comment:
From TDB 335.61 :
```sql
SELECT * FROM `gossip_menu_option` WHERE `menu_id` IN (10638,10639);
```
```
menu_id      id  option_icon  option_text                           OptionBroadcastTextID  option_id  npc_option_npcflag  action_menu_id  action_poi_id  box_coded  box_money  box_text  BoxBroadcastTextID  VerifiedBuild
-------  ------  -----------  ------------------------------------  ---------------------  ---------  ------------------  --------------  -------------  ---------  ---------  --------  ------------------  -------------
  10638       0            0  I no longer wish to gain experience.                  35531          1                   1               0              0          0          0                             0              0
```
Used in the script like this:
```cpp
+{
 +    OPTION_ID_STOP_GAINING_XP  = 0,
 +    OPTION_ID_START_GAINING_XP = 1,
 +    XP_GAIN_STOP_START_COST    = 100000 //10 00 00 copper (10 gold)
 +};
```
But I am just about to modify it more or less back to what it was (except for the hardcoded text, of course).
I need to use the script check to produce a condition to see if player can gain experience or not and based on that result, choose which one of the 2 gossip options to use:
```cpp
bool noXPGain = player->HasFlag(PLAYER_FLAGS, PLAYER_FLAGS_NO_XP_GAIN);
```